### PR TITLE
httpdiff 1.0.0 (import from head-only)

### PR DESCRIPTION
**Pull request by bfontaine**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

https://github.com/Homebrew/homebrew-head-only/pull/220

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
👍

File: Formula/httpdiff.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/jgrahamc/httpdiff"
+  url "https://github.com/jgrahamc/httpdiff/archive/v1.0.0.tar.gz"
+  sha256 "b2d3ed4c8a31c0b060c61bd504cff3b67cd23f0da8bde00acd1bfba018830f7f"
+
+  head "https://github.com/jgrahamc/httpdiff.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/httpdiff"
```
Comment (zmwangx) [inline Formula/httpdiff.rb]: `bin/"httpdiff"` should be better.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by MikeMcQuaid** [comment]
Thanks @bfontaine!

**Comment by bfontaine** [comment]
Sorry for the delay @MikeMcQuaid; I couldn’t get the bottle right.

**Pull request closed without merging**